### PR TITLE
Add github action

### PR DESCRIPTION
**Pull request by hbinkim**
## Done in this PR
- [x] golang lint
- [x] go mod verify check
- [x] manifests: controller-gen check
- [x] unit test
- [x] docker image build
- [x] auto assign reviewers
- [x] auto size label
- [x] make local docker registry

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,3 +83,38 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
...
+
+minikube-clean:
+	CHANGE_MINIKUBE_NONE_USER=true sudo -E minikube delete
+
+# Kubebuilder related command
+kubebuilder-download:
+	curl -L https://go.kubebuilder.io/dl/2.3.1/linux/amd64 | tar -xz -C /tmp/
+	sudo mv /tmp/kubebuilder_2.3.1_linux_amd64 /usr/local/kubebuilder
+	export PATH=$(PATH):/usr/local/kubebuilder/bin
+
```
Comment (hyoung-90) [inline Makefile:120]: Please add `make clean` to cleanup after `make deploy` command.
```
clean:
	kustomize build config/default | kubectl delete -f -
```

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= hypersds-operator:latest
```
Comment (hyoung-90) [inline Makefile:3]: It seems that registry information should be included for `docker-push`.

File: Makefile (hunk 1 above)
Comment (hbinkim) [reply Makefile:124]: Added :D

File: Makefile (hunk 2 above)
Comment (hbinkim) [reply Makefile:3]: Added on the todo list for the next PR

**Pull request merged**